docs/pages/guides/adopting-prebuild.md

```diff
@@ -0,0 +1,128 @@
+
+## Bootstrap a new project
+
+<!-- NOTE: Update the version when we bump support -->
+
+<Terminal cmd={["$ npx react-native init --version 0.69.4"]} />
+
+This will create a new React Native project. Note that not all versions of `react-native` are supported, be sure to use a version of `react-native` that has a corresponding Expo SDK version.
+
+## Install the `expo` package
+
+The `expo` package contains the `npx expo prebuild` command and indicates which prebuild template to use:
+
+<Terminal cmd={["$ yarn add expo"]} />
+
+Ensure you install the version of `expo` that works for your currently installed version of `react-native`. Note that not all major versions of `react-native` are supported.
+
+## Update the entry file
+
+Modify the entry file like so:
+
+```diff
++ import {registerRootComponent} from 'expo';
+
+- import {AppRegistry} from 'react-native';
+import App from './App';
+- import {name as appName} from './app.json';
+
+- AppRegistry.registerComponent(appName, () => App);
++ registerRootComponent(App);
+```
+
+Alternatively, you can just change `appName` to `"main"`. Switching to `registerRootComponent` is useful if you want to take advantage of other features like web support. Importing `expo` adds side-effects for development environments that are removed in production bundles, meaning there isn't some massive bundle size increase.
+
+## Prebuild
+
+> **Warning:** Make sure you've committed your changes in case you want to revert, the command will warn you about this too!
+
+Run the following command to regenerate the `/ios` and `/android` directories based on the `app.json` configuration.
+
+<Terminal cmd={[
+'$ npx expo prebuild --clean',
+]} cmdCopy="npx expo prebuild --clean" />
+
+You can test that everything worked by building the projects locally:
+
+<Terminal cmd={[
+'# Build your native Android project',
+'$ npx expo run:android',
+'',
+'# Build your native iOS project',
+'$ npx expo run:ios'
+]} />
+
+## Extra changes
+
+The following changes are optional but recommended.
+
+**.gitignore**
+
+You can add `.expo/` to your `.gitignore` to prevent generated values from Expo CLI from being committed. You can also add `ios/` and `android/` to the `.gitignore` if you want to ensure they aren't committed between prebuilds.
+
+**app.json**
+
+Remove all fields that are outside of the top-level `expo` object as these will not be used in `npx expo prebuild`.
+
+```diff
+{
+-  "name": "myapp",
+-  "displayName": "myapp"
++  "expo": {
++    "name": "myapp"
++  }
+}
+```
+
+**metro.config.js**
+
+- See [Customizing Metro](/guides/customizing-metro.md)
+
+**package.json**
+
+You may want to change the scripts to use the Expo CLI run commands:
+
+```diff
+  "scripts": {
+    "start": "expo start --dev-client",
+-    "android": "react-native run-android",
+-    "ios": "react-native run-ios",
++    "android": "expo run:android",
++    "ios": "expo run:ios",
+  },
+```
+
```

Comment:
should we add babel config too?

Reply:
Babel config is required for web support and OTA updates, it doesn't overlap with `prebuild` afaict.